.editorconfig

```
@@ -0,0 +1,23 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{rs,toml}]
+indent_style = space
+indent_size = 4
+
+[*.lua]
+indent_style = tab
+indent_size = 2
```

```suggestion
indent_size = 4
```
inline with stylua defaults